src/tests/java17/BigIntegerTest.java

```diff
@@ -0,0 +1,16 @@
+package java17;
```

Comment:
Please put the new tests into the namespace of the regular tests, as they enhance tests on existing features rather than covering something that was introduced with Java 17.